week-2/implement/cases.js

```diff
@@ -15,3 +15,12 @@
 
 // Come up with a clear, simple name for the function
 // Use the string documentation to help you plan your solution
+
+function upperCamelCase(text) {
+  let upperCamelCaseText = text.replaceAll(" ", "_").toUpperCase();
+  return upperCamelCaseText;
```

Comment:
That's a great solution 💯 . Also, the function name has a mention of the snake _ case as you make Camel and Snake case.